lib/index.js

```diff
@@ -21,25 +21,14 @@ function inspect(root, targetFile, options) {
     {cwd: root}
   )
     .then(function (result) {
-      var parseResult = {};
-      try {
-        parseResult = parse(result, options.dev);
-      } catch (error) {
-        console.log('\nAn unknown error occurred. ' +
-      'Please include the trace below when reporting to Snyk:', error, '\n');
-        return Promise.reject('');
-      }
-      if (parseResult.ok) {
-        return {
-          plugin: {
-            name: 'bundled:maven',
-            runtime: 'unknown',
-          },
-          package: parseResult.data,
-        };
-      }
-      return Promise.reject(parseResult.message ||
-      'An internal error has occured. Please contact Snyk for support.');
+      var parseResult = parse(result, options.dev);
+      return {
+        plugin: {
+          name: 'bundled:maven',
+          runtime: 'unknown',
+        },
+        package: parseResult.data,
+      };
```

Comment:
Removing all this fancy error handling here in favour of `Error` objects thrown anywhere 'below' in code.